### ISSUE TITLE
Day 1: cross_validation was deprecated.

### ISSUE THREAD

**Issue by cyyeap** [Avik-Jain/100-Days-Of-ML-Code#36]
Warning (from warnings module):
  File "/usr/local/lib/python2.7/dist-packages/sklearn/cross_validation.py", line 44
    "This module will be removed in 0.20.", DeprecationWarning)
DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.

Should we update from sklearn.cross_validation import train_test_split to
from sklearn.model_selection import train_test_split?

**Comment by mounicanaidu** [comment]
Yes, it is advisable to start using sklearn.model_selection instead of sklearn.cross_validation as it will removed in the future releases for creating train and test splits.

**Comment by aaaazAAR** [comment]
Sure

**Comment by sid321axn** [comment]
**Problem:**
The deprecation warning means that the module is deprecated, i.e. no longer supported. You are using a version for which sklearn.cross_validation is not a module any longer.
**Solution:**
from sklearn.model_selection import train_test_split
**Reference :** [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html)

**Warning:** DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20. "This module will be removed in 0.20.", DeprecationWarning)
